docs/content/manual/rancher-integration/drain-using-rancher-ui.md

```diff
@@ -0,0 +1,211 @@
+
+**Given**  Single node cluster with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND 1 RWO and 1 RWX volumes unattached
+
+**When** Drain the node with default values of Rancher UI
+
+**Then** Drain should be successful
+
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors
+
+
+### 2. Drain operation on a multiple node using Rancher UI - default value
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND few RWO and RWX volumes unattached
+
+**When** Drain the worker node one by one with default values of Rancher UI
+
+**Then** Drain should be successful
+ 
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors
+
+
+### 3. Drain operation on a multiple node using Rancher UI - Force
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND few RWO and RWX volumes unattached
+
+**When** Drain the worker node one by one with Force option enabled using Rancher UI.
+
+**Then** Drain should be successful
+
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors.
+
+
+### 4. Drain operation on a multiple node using Rancher UI - Single replica of volumes
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes with 1 replica attached with node/pod exists
+
+AND few RWO and RWX volumes with 1 replica unattached
+
+**When** Drain the worker node one by one, start with the node which has volume's replica
+
+**Then** Drain should be successful
+
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors
+
+
+### 5. Longhorn volume with 50/100 Gi data
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND 1 volume with 50/100 Gi of data attached to a node
+
+**When** Drain the node using Rancher UI with default value 
+
+**Then** Drain should be successful for 2 node but should fail for last node
+
+AND Volume data should be intact
+
+AND Once there are 2 healthy replica rebuilt, then retry drain should be successful
```

Comment:
Why there will be 2 healthy replicas after we've drained 2 nodes and only 1 node left?

Reply:
I mean once there are 2 healthy replicas eventually on the nodes which got drained successfully, then retry drain should be successful on the last node.

Reply:
I need to enable both `Delete Empty Dir Data` and `Force` to drain node which volume not attached to so the node drain success, when I try drain node which volume attached to, the drain stuck ( there still exist another available node in cluster)

Reply:
The draining should success on the node where the volume is not attached as far I know. Let's discuss this offline.

Reply:
Hi @khushboo-rancher ,
For this test case, enable `Delete Empty Dir Data` is enough and test result [here](https://github.com/longhorn/longhorn/issues/5708#issuecomment-1498770558) for your reference.